### PR TITLE
feat(docs) adding xmas newsletter

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  alt="An image showing Santa's SELECT clause"
+  height={467}
+  src="/img/blog/2022-12-19/banner.png"
+  width={650}
+></Banner>
+
+
+## Merry Questmas!
+
+We wishes you a year full of joy, peace, and prosperity. We will keep your
```
Comment (amyshwang) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:32]:
```suggestion
We wish you a year full of joy, peace, and prosperity. We will keep your
```

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+We wish you a year full of joy, peace, and prosperity. We will keep your
+ingestion speedy and your query swift.
+
+## What's coming for 2023
+
+In the meantime, look out for the QuestDB Santa's gifts:
+
+- Support for parallel ingestion across all protocols via WAL
+- Pandas integration
+- UUID
```
Comment (javier) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:41]: Is this a new data type? If so maybe "New UUID data type", otherwhise maybe "Native support for UUIDs"

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Explore the power of QuestDB on our [demo site](https://demo.questdb.io/), and
```
Comment (javier) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:43]: Maybe to make it super explicit we could say "Explore the power of QuestDB by running the example queries on our demo site"

Comment (amyshwang) [reply blog/2022-12-19-merry-questmas-gifts-2023.mdx:43]:
```suggestion
Explore the power of QuestDB by running the example queries on our [demo site](https://demo.questdb.io/), and
```

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx (hunk 2 above)
Comment (jerrinot) [reply blog/2022-12-19-merry-questmas-gifts-2023.mdx:41]: yes, it's a new data type.

Comment (amyshwang) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:41]:
```suggestion
- New `UUID` data type
```

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+## Merry Questmas!
+
+We wish you a year full of joy, peace, and prosperity. We will keep your
+ingestion speedy and your query swift.
+
+## What's coming for 2023
+
+In the meantime, look out for the QuestDB Santa's gifts:
+
+- Support for parallel ingestion across all protocols via WAL
```
Comment (amyshwang) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:39]:
```suggestion
- Parallel ingestion across all protocols via WAL
```

File: blog/2022-12-19-merry-questmas-gifts-2023.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  width={650}
+></Banner>
+
+
+## Merry Questmas!
+
+We wish you a year full of joy, peace, and prosperity. We will keep your
+ingestion speedy and your query swift.
+
+## What's coming for 2023
```
Comment (amyshwang) [inline blog/2022-12-19-merry-questmas-gifts-2023.mdx:35]:
```suggestion
## What's coming for January 2023
```

**Review by javier — APPROVED** [review]
Looks good to me!

**Pull request merged**